src/static/js/routes/careers/working-at-cfpb/index.js

```diff
@@ -0,0 +1,21 @@
+/* ==========================================================================
```

Comment:
Because when per-page JS is implemented we don't want to be loading the Slick carousel code on every page. If we just have a few pages that have carousels they will have the same boilerplate initialization on the page-specific scripts, which I think is fine. If we start getting an unwieldy number of carousels we can move that initialization over to common.js to load it on all pages.